app/assets/javascripts/spotlight/block_mixins/previewable.js

```diff
@@ -0,0 +1,101 @@
+(function ($){
+  SirTrevor.PreviewButton = function() {
+    this._ensureElement();
+    this._bindFunctions();
+  }
+
+  SirTrevor.PreviewButton.prototype = Object.create(SirTrevor.BlockDeletion.prototype);
```

Comment:
This is annoying, but SirTrevor doesn't expose the base class we actually want..